iOS/Services/StitchLocalMongoDBService/StitchLocalMongoDBService/LocalMongoClient.swift

```diff
@@ -165,6 +153,6 @@ private final class MongoDBServiceClientProvider: ThrowingServiceClientProvider
 }
 
 /// MongoDBService singleton
-public let MongoDBService = AnyThrowingServiceClientProvider<MongoClient>.init(
-    provider: MongoDBServiceClientProvider()
+public let mongoDBService = AnyThrowingServiceClientFactory<MongoClient>.init(
```

Comment:
[nit] this should probably be called `localMongoDBServiceClientFactory` to be clear that it's a factory and to be clear that it's for local MongoDB